LoopTests/Models/Remote/BolusActionTests.swift

```diff
@@ -1,101 +0,0 @@
-//
```

Comment:
Unfortunately these tests I recently added are not easily supported now with these changes. I may try to add these back with some architecture improvements later.